stub.c

```diff
@@ -155,6 +167,11 @@ static EFI_STATUS run(EFI_HANDLE image) {
          * as potential command line to use. */
         (void) process_arguments(image, loaded_image, &cmdline);
         parse_cmdline(cmdline);
+        if (log_isdebug == true) {
+                log_debug("Stubble configuration:");
+                log_debug("debug: on");
+                log_debug("dtb_override: %s", dtb_override ? "on" : "off");
```

Comment:
Maybe better use "enabled" and "disabled" instead of "on" and "off" for all options.